src/dlls/mscoree/coreclr/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ else()
 
 endif (WIN32)
 
+set(CMAKE_ADDITIONAL_LINKEROPTIONS "" )
```

Comment:
Could you please remove the CMAKE_ prefix from the variable name? This prefix is used by cmake for its own variables.